edx_repo_tools/codemods/node16/gha_ci_modernizer.py

```diff
@@ -0,0 +1,81 @@
+"""
+Node Modernizer for Github Actions CI
+"""
+import click
+from copy import deepcopy
+
+from edx_repo_tools.utils import YamlLoader
+
+ALLOWED_NODE_VERSIONS = [12, 14, 16]
+ALLOWED_NPM_VERSION = '8.x.x'
+
+
+class GithubCINodeModernizer(YamlLoader):
```

Comment:
the class name should be `GithubCiNodeModernizer` as per convention or something like `NodeWorkflowModernizer`